lib/cmd/license-cmd.js

```diff
@@ -3,28 +3,31 @@
 
 const program = require('commander');
 const ThirdPartyLicenseGenerator = require('./third-party-license-generator');
+const LicenseFileHandler = require('./license-file-handler');
 
 program
-  .option('--header <value>', 'Add a header to license file.')
-  .option('--allow <value>', 'Comma separated list of allowed license names to be verified.')
-  .option('--file <value>', 'File to write the license list.')
-  .option('--skipPrefix <value>', 'Skip licenses of dependencies that start with this')
+  .option('--config <value>', 'Use the config file.')
   .parse(process.argv);
 
-let header = program.header;
-let allowedLicenseList = program.allow.split(',');
-let file = program.file;
-let skipPrefix = program.skipPrefix;
+let config = null;
+
+LicenseFileHandler
+  .getFile(program.config ? program.config : '.license-config')
```

Comment:
If anyone accidentally change '.license-config' to 'license-config' here, all tests will pass.
Why did you tested config here? You can't test it inside getFile()? For example:
```
const defaultConfigFileName = '.license-config';
getFile(configFileName){
if(!configFile){
configFile = defaultConfigFileName
}
//..the method getFile
}
```